apps/src/componentLibrary/dropdown/simpleDropdown/simpleDropdown.module.scss

```diff
@@ -209,7 +260,7 @@
 
   .dropdownLabel {
     @include label-one;
-    margin-bottom: 0.37rem;
+    margin-bottom: 0;
```

Comment:
Where do we take these values from?

Reply:
TextField was my guide on these, since they'll be mixed together in the same form in most cases